dap-dlv-go.el

```diff
@@ -93,6 +99,18 @@
       (plist-put conf :mode "test")
     (plist-put conf :mode "debug")))
 
+(defun dap-dlv-go--populate-test-args (conf)
+  "Populate auto arguments according to CONF."
+  (dap--put-if-absent conf :program "."))
+
+(defun dap-dlv-go--extract-current-function-name ()
```

Comment:
The alternative is to use `textDocument/symbols` similar to what dap-java.el do. I would say that it will return better results.

Reply:
Updated